exec/java-exec/src/test/java/org/apache/drill/exec/testing/TestExceptionInjection.java

```diff
@@ -216,79 +216,77 @@ public void injectionOnSpecificBit() {
     final ZookeeperHelper zkHelper = new ZookeeperHelper();
     zkHelper.startZookeeper(1);
 
-    // Creating two drillbits
-    final Drillbit drillbit1, drillbit2;
-    final DrillConfig drillConfig = zkHelper.getConfig();
     try {
-      drillbit1 = Drillbit.start(drillConfig, remoteServiceSet);
-      drillbit2 = Drillbit.start(drillConfig, remoteServiceSet);
-    } catch (DrillbitStartupException e) {
-      throw new RuntimeException("Failed to start drillbits.", e);
-    }
+      // Creating two drillbits
+      final Drillbit drillbit1, drillbit2;
+      final DrillConfig drillConfig = zkHelper.getConfig();
+      try {
+        drillbit1 = Drillbit.start(drillConfig, remoteServiceSet);
+        drillbit2 = Drillbit.start(drillConfig, remoteServiceSet);
+      } catch (DrillbitStartupException e) {
+        throw new RuntimeException("Failed to start drillbits.", e);
+      }
 
-    final DrillbitContext drillbitContext1 = drillbit1.getContext();
-    final DrillbitContext drillbitContext2 = drillbit2.getContext();
+      final DrillbitContext drillbitContext1 = drillbit1.getContext();
+      final DrillbitContext drillbitContext2 = drillbit2.getContext();
 
-    final UserSession session = UserSession.Builder.newBuilder()
-        .withCredentials(UserBitShared.UserCredentials.newBuilder().setUserName("foo").build())
-        .withUserProperties(UserProperties.getDefaultInstance())
-        .withOptionManager(drillbitContext1.getOptionManager())
-        .build();
+      final UserSession session = UserSession.Builder.newBuilder().withCredentials(UserBitShared.UserCredentials.newBuilder().setUserName("foo").build()).withUserProperties(UserProperties.getDefaultInstance()).withOptionManager(drillbitContext1.getOptionManager()).build();
```

Comment:
Did we want to combine all the method calls onto one line rather than the fluent, multi-line style originally in the code? An artifact of a code formatter?

Reply:
I'm a victim of autoformat. Fixed.